py_gtfs_rt_ingestion/tests/test_gtfs_rt_converter.py

```diff
@@ -16,19 +16,18 @@ def test_bad_conversion_local() -> None:
     be added to the error files list
     """
     # dummy config to avoid mypy errors
-    converter = GtfsRtConverter(config_type=ConfigType.RT_ALERTS)
+    converter = GtfsRtConverter(
```

Comment:
all of this uses the `process_files` method rather than `convert` since we don't want to write to s3, nor archive files.